dex/testing/loadbot/mantle.go

```diff
@@ -110,7 +110,8 @@ out:
 		}
 	}
 
-	m.ForceShutdown()
+	// Let Core shutdown and lock up.
+	m.waiter.WaitForShutdown()
```

Comment:
I haven't retested loadbot with this, but `ForceShutdown` was the wallet and account locking part of `PromptPass` that is now done by `Core` shutdown.